### PR TITLE
Add Nazarè 1-60 PCB support

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description
Adds support for the Nazarè 1-60 PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nazare160/alpha/config.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define RGB_DI_PIN B15
+#define RGBLED_NUM 10
+#define RGBLIGT_LIMIT_VAL 200
+#define RGBLIGHT_SLEEP
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/nazare160/alpha/config.h:47]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/nazare160/alpha/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = yes    # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/nazare160/alpha/rules.mk:15]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/nazare160/alpha/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = STM32F072
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes    # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/nazare160/alpha/rules.mk:7]:
```suggestion
BOOTMAGIC_ENABLE = yes    # Enable Bootmagic Lite
```

File: keyboards/nazare160/alpha/chconf.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define CH_CFG_ST_FREQUENCY 10000
```
Comment (drashna) [inline keyboards/nazare160/alpha/chconf.h:19]: Is this needed?  

If not, please remove.

File: keyboards/nazare160/beta/chconf.h [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define CH_CFG_ST_FREQUENCY 10000
```
Comment (drashna) [inline keyboards/nazare160/beta/chconf.h:19]: Is this file needed?

File: keyboards/nazare160/beta/rules.mk [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/nazare160/beta/rules.mk:17]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/nazare160/readme.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Nazarè Engineering Nazare1-60 QMK Firmware
+
+![nz160](https://i.imgur.com/ZYJgL2G.jpg)
```
Comment (drashna) [inline keyboards/nazare160/readme.md:3]:
```suggestion
![nz160](https://i.imgur.com/ZYJgL2Gh.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/nazare160/alpha/config.h (hunk 1 above)
Comment (Gondolindrim) [reply keyboards/nazare160/alpha/config.h]: Applied in [b3cf6fa](https://github.com/qmk/qmk_firmware/pull/16822/commits/b3cf6fa608c83ade5d3d63a8263a0862c2b4348a)

File: keyboards/nazare160/beta/chconf.h (hunk 5 above)
Comment (Gondolindrim) [reply keyboards/nazare160/beta/chconf.h]: Removed in f9311d0

File: keyboards/nazare160/alpha/chconf.h (hunk 4 above)
Comment (Gondolindrim) [reply keyboards/nazare160/alpha/chconf.h]: Removed in f9311d0

File: keyboards/nazare160/alpha/rules.mk (hunk 3 above)
Comment (Gondolindrim) [reply keyboards/nazare160/alpha/rules.mk]: Committed in e7cade2

File: keyboards/nazare160/beta/rules.mk (hunk 6 above)
Comment (Gondolindrim) [reply keyboards/nazare160/beta/rules.mk]: Committed in [e3ad625](https://github.com/qmk/qmk_firmware/pull/16822/commits/e3ad625cede651fae12742958d2561a6f0c89f0e)

File: keyboards/nazare160/readme.md (hunk 7 above)
Comment (Gondolindrim) [reply keyboards/nazare160/readme.md]: Committed in [19bfe4f](https://github.com/qmk/qmk_firmware/pull/16822/commits/19bfe4fe5844156b33971ff8bc92bc6e5e3c7b20)

**Review by drashna — APPROVED** [review]

File: keyboards/nazare160/beta/rules.mk [hunk 8]
```diff
@@ -0,0 +1,26 @@
+MCU = STM32F411
```
Comment (fauxpark) [inline keyboards/nazare160/beta/rules.mk:1]:
```suggestion
# MCU name
MCU = STM32F411
```

File: keyboards/nazare160/alpha/rules.mk [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = STM32F072
```
Comment (fauxpark) [inline keyboards/nazare160/alpha/rules.mk:2]:
```suggestion
MCU = STM32F072

# Bootloader selection
BOOTLOADER = stm32-dfu
```

File: keyboards/nazare160/alpha/rules.mk [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+#
+BOOTMAGIC_ENABLE = yes    # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/nazare160/alpha/rules.mk:15]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/nazare160/beta/rules.mk (hunk 8 above)
Comment (Gondolindrim) [reply keyboards/nazare160/beta/rules.mk:2]: Committed in [e2895d8](https://github.com/qmk/qmk_firmware/pull/16822/commits/e2895d81e81b944b3f008b0ffa48599be08d54bd)

File: keyboards/nazare160/alpha/rules.mk (hunk 9 above)
Comment (Gondolindrim) [reply keyboards/nazare160/alpha/rules.mk:2]: Committed in [bd52d50](https://github.com/qmk/qmk_firmware/pull/16822/commits/bd52d5069c706a3cc4cf27fe0dcff833774e1176)

File: keyboards/nazare160/alpha/rules.mk (hunk 10 above)
Comment (Gondolindrim) [reply keyboards/nazare160/alpha/rules.mk]: Committed in [7d9c744](https://github.com/qmk/qmk_firmware/pull/16822/commits/7d9c7440ad8dbcccb2440dbec64d569ba84d68de)

File: keyboards/nazare160/beta/rules.mk [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+WS2812_DRIVER = pwm
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = no
```
Comment (fauxpark) [inline keyboards/nazare160/beta/rules.mk:21]:
```suggestion
```

File: keyboards/nazare160/beta/rules.mk [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+EEPROM_DRIVER = i2c
+
+ENCODER_ENABLE = no
```
Comment (fauxpark) [inline keyboards/nazare160/beta/rules.mk:25]:
```suggestion
```

File: keyboards/nazare160/alpha/rules.mk [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+BOOTMAGIC_ENABLE = yes    # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = no
```
Comment (fauxpark) [inline keyboards/nazare160/alpha/rules.mk:19]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Missing info.json
